### PR TITLE
Folder with new pictures to test the accuracy of our predictions. The…

### PR DESCRIPTION
**Pull request by lady-hamster**
… pictures are from unrepresented labels and the authors are not in the database. The folder contains a csv file with the correct genre (I tried to mimick the fields of the original database).

### CODE REVIEW

**Comment by michaelsmueller** [comment]
One thing would be good is to ensure that these images are similar to our test images used in training. For example, our training data uses `.jpg` images only, they are resized to 224x224, with RGB, etc.

But in this performance test set, I can see we have some `.png` files, some larger images (e.g. `7.png` is 838 x 789 pixels), etc. Also we might standardize the naming (Firstname_Lastname_#.jpg)
